### DISCUSSION TITLE
npm install is not working

### DISCUSSION THREAD

**Discussion by renep16** [shadowwalker/next-pwa#176] [Q&A]
Hi, I am creating a new project, and before has used this package getting awesome results.
Now with this new project I run the npm i next-pwa on terminal on windows and I am getting the next issue
`npm ERR! Unexpected end of JSON input while parsing near '...Zv35X6UHh2dLW5ydvh08x'`

Have try differents ways to get the package install it and I can't. 
If any solution I will appreciate it, really love the project and package

**Comment by yuxxeun** [comment]
u can try with `yarn add next-pwa`
